commands/src/main/java/org/wildfly/extras/creaper/commands/logging/HandlerType.java

```diff
@@ -0,0 +1,22 @@
+package org.wildfly.extras.creaper.commands.logging;
+
+/**
+ * @author Ivan Straka istraka@redhat.com
+ */
+public enum HandlerType {
+
+    CONSOLE("console-handler"),
+    PERIODIC("periodic-rotating-file-handler");
```

Comment:
There are at least two important properties of this log handler -- that it's file-based and that it uses periodic log rotation. I'd call it `FILE_ROTATING` or something like that.